spiketoolkit/postprocessing/postprocessing_tools.py

```diff
@@ -365,8 +366,9 @@ def get_unit_max_channel(recording, sorting, unit_ids=None, grouping_property=No
                                        ms_before=ms_before, ms_after=ms_after,  grouping_property=grouping_property,
                                        compute_property_from_recording=compute_property_from_recording,
                                        verbose=verbose)
-        max_channel = np.unravel_index(np.argmax(np.abs(template)),
-                                       template.shape)[0]
+        max_chan = np.unravel_index(np.argmax(np.abs(template)),
+                                    template.shape)[0]
+        max_channel = recording.get_channel_ids()[max_chan]
```

Comment:
Here I would explicitly use **max_chan_index** and **max_chan_id** in the code in case of the variable will be used further.

Reply:
ok